### ISSUE TITLE
Not an issue, but a big ole Thanks!!

### ISSUE THREAD

**Issue by wizaerd** [kevinchappell/formBuilder#391]
First the compliment...  Thank you so much for putting this together.  I had been looking for a client side form creation method for a while now, having looked at quite a few of them before settling on this one.  It's really a great module/plugin/app...

But I also see you're working on a v2.0, Formeo? Should I be abandoning this and moving to the Formeo? I really don't need the backend services of the form saving, collecting values, etc... I've actually already implemented my own processes for that.  But I do want to make use of the new features and bug fixes as it relates to the UI aspect.

**Comment by kevinchappell** [comment]
tl; dr
Formeo

looks like this is where I outline the master plan. i'm actually preparing 2 major releases.

**formBuilder 2.0**
- integrated i18n (give it a locale, it loads the language)
- custom subtypes, templates, onRender callbacks for fields etc
- html labels and wysiwyg editing
- nuuumerous bugfixes.
- many optimizations
- final major release

Originally planned as the 1.25 release, formBuilder 2.0 makes sense considering the number of changes, some of them major. Once released only critical bugs will be patched- no new features and once Formeo 1.0 releases- formBuilder will be officially deprecated. The 3 major reasons for this are:
1. The changes needed to implement all of the requested features to formBuilder would make it incompatible with data saved from previous versions.
2. I spend literally most of my day supporting formBuilder, its consuming too much time. Further development would mean breaking changes which means more support and my bandwidth is already maxed. There is no way I can support both.
3. Formeo is better than formBuilder in every way- performance, customization, API... everything.

**Formeo 1.0**
- pure Javascript- no dependencies
- robust api; interact with and control any part of the module
- customization: custom elements, layouts, templates, events, actions
- repeatable input groups and form, stage, row, column and field settings
- columns/inline-fields (a daily request in formBuilder)
- i18n support thanks to all of the generous contributions from @HayriCan, @ryandiv, @xoniq, @J2TeaM, @dragosrobertn, @alien-art, @dniki, @AcNeidert, @hasanbozok, @pooya-eghbali, @bit950, @Krato, and @YI-CHING-WU

So to answer your question; if *I* was building a new app that needed a form editor, I would work on the non-editor parts while waiting for Formeo 1.0. I am currently writing documentation and making minor adjustments but all of the major pieces are in place now. The last 7 months have been slow but these releases are finally on the horizon.

### and now...
A big thanks to you and the entire community for your patience and support in development. As cheesy as it sounds, without you, formBuilder wouldn't be where it is today. Every issue and email has helped make formBuilder and ultimately Formeo a better product.

**Comment by nk-nareshkumar** [comment]
@wizaerd  kindly share your email id or skype.. my skype phpdevelopernk ..

**Comment by wizaerd** [comment]
@nk-nareshkumar my email is wizaerd at gmail.com

**Comment by ankurk91** [comment]
https://github.com/Draggable/formeo
throws a 404, seems like now it is a private repo.
